moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -124,8 +124,10 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     goal_orientation_tolerance_ = 1e-3;  // ~0.1 deg
     allowed_planning_time_ = 5.0;
     num_planning_attempts_ = 1;
-    max_velocity_scaling_factor_ = 1.0;
-    max_acceleration_scaling_factor_ = 1.0;
+    node_handle_.param<double>("robot_description_planning/joint_limits/default_velocity_scaling_factor",
+                               max_velocity_scaling_factor_, 0.1);
+    node_handle_.param<double>("robot_description_planning/joint_limits/default_acceleration_scaling_factor",
+                               max_acceleration_scaling_factor_, 0.1);
```

Comment:
There is quite a number of places with hard-coded `0.1` now, but I'm not sure we have a nice place to define a constant for this.

`0.1` seems overly slow to me though, I would have picked `0.5` myself.
Maybe we want to haggle over the exact number in the maintainer meeting next week?

Reply:
I think we mostly need to decide which situation the default value should be for. In my opinion, it should be a safe ballpen mode for new users, which leaves them enough time to hit the emergency switch (which they will invariably not have in their hand) in case they accidentally send a motion request that does not do what they expected. E.g. the IK flipped and the robot is doing a large motion, or the robot hits something that wasn't represented in the environment (like the robot base), etc.

Choosing a "very safe" default value and expecting the user to adjust the speed to when they are ready is probably better than the opposite.

I tested a few parameter values with our UR5 to check which speeds would still be surprising a user in this type of situation. `0.5` is barely different from full speed for small motions, `0.2` seemed to be still pretty fast, and `0.1` seemed safe. `0.1` for a UR5 is about `10 cm/s` in most configurations, which I think this is a good compromise.